psi4/driver/procrouting/sapt/sapt_util.py

```diff
@@ -143,21 +143,37 @@ def print_sapt_dft_summary(data, name, short=False):
     ret += "\n"
     core.set_variable("SAPT IND ENERGY", ind)
 
+
+    # Exchange-dispersion scaling
+    exch_disp_scheme = core.get_option("SAPT", "SAPT_DFT_EXCH_DISP_SCALE_SCHEME")
+    if exch_disp_scheme == "NONE":
+        data["Exch-Disp20,r"] = data["Exch-Disp20,u"] 
+    if exch_disp_scheme == "FIXED":
+        exch_disp_scale = core.get_option("SAPT", "SAPT_DFT_EXCH_DISP_FIXED_SCALE")
+        data["Exch-Disp20,r"] = exch_disp_scale * data["Exch-Disp20,u"]
+    if exch_disp_scheme == "DISP":
+        exch_disp_scale = data["Disp20"] / data["Disp20,u"]
+        data["Exch-Disp20,r"] = exch_disp_scale * data["Exch-Disp20,u"]
+    
+
     # Dispersion
-    disp = data["Disp20"] + data["Exch-Disp20,u"]
+    # disp = data["Disp20"] + data["Exch-Disp20,u"]
+    disp = data["Disp20"] + data["Exch-Disp20,r"]
     ret += print_sapt_var("Dispersion", disp) + "\n"
     ret += print_sapt_var("  Disp2,r", data["Disp20"]) + "\n"
     ret += print_sapt_var("  Disp2,u", data["Disp20,u"]) + "\n"
+    if exch_disp_scheme != "NONE":
+        ret += print_sapt_var("  Exch-Disp2,r (scaled)", data["Exch-Disp20,r"]) + "\n"
```

Comment:
```suggestion
        ret += print_sapt_var("  Est. Exch-Disp2,r", data["Exch-Disp20,r"]) + "\n"
```

Looks from libsapt_solver that `Est.` is the usual notation